src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -130,6 +135,31 @@ class EmojiPickerMenu extends Component {
         this.setState({selection: event.nativeEvent.selection});
     }
 
+    /**
+     * Calculate the initial filtered emojis and header row indices
+     * @returns {Object}
+     */
+    getInitialFilteredEmojisAndHeaderRowIndices() {
```

Comment:
As this method is not only calculated the emojis and indices on initial render, can we change the name to be more appropriate?

Reply:
imo remove `initial` from it and should be fine 😆